impeller/aiks/aiks_playground.cc

```diff
@@ -14,19 +15,30 @@ AiksPlayground::AiksPlayground() = default;
 
 AiksPlayground::~AiksPlayground() = default;
 
-bool AiksPlayground::OpenPlaygroundHere(const Picture& picture) {
+bool AiksPlayground::OpenPlaygroundHere(
+    const Picture& picture,
+    std::unique_ptr<TextRenderContext> text_render_context_override) {
```

Comment:
Instead of the context being provided as as override to each playground open call, I'd prefer the fixture itself take and store the context. Does it need to be unique ptr? If not, it could just store it as a shared ptr.

Avoid having to patch each call-site to open the playground and looks more extensible.

Reply:
By the "fixture" storing the context, do you mean having something like `AiksPlayground::SetTextRenderContext()`?

Reply:
Yes.

Reply:
SGTM, done.